src/python/tests/core/build_management/revisions_test.py

```diff
@@ -347,6 +347,30 @@ def test_get_component_range_list_custom_binary_job(
                      revisions.get_component_range_list(1337, 1338,
                                                         CUSTOM_BINARY_JOB_TYPE))
 
+  def test_find_build_url(self):
```

Comment:
Not related to fix, but noticed a missing test in seemingly related code, so adding.